### PR TITLE
Add noderunner.md

### PR DESCRIPTION
**Pull request by vmapetr**
This PR adds noderunner.md doc that explains how to manually install an outdated Node 6 runner.

### CODE REVIEW

File: docs/noderunner.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+Use the following pipeline task sample to install the latest version of Node 6 runner:
+
+```yaml
+  - task: NodeTaskRunnerInstaller@0
+    inputs:
+      runnerVersion: 6
+```
+
+Please, check more details in [NodeTaskRunnerInstaller task](TODO) documentation.
```
Comment (max-zaytsev) [inline docs/noderunner.md:48]: `TODO` should be visible to users

File: docs/noderunner.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Since Node 6 has long passed out of the upstream maintenance window, and all officially supported tasks are migrated from Node 6 to Node 10, Node 6 soon will be removed from the agent package. 
+It's also highly recommended to third-party task maintainers migrate tasks to Node 10 or Node 16.
+
+However, to support backward compatibility with the Node 6 tasks we provide self-service methods to install the designated Node runner manually.
+
+## Install Node 6 runner manually
+
+To support the execution of Node 6 tasks agent should be provided with the latest Node 6 version - `6.17.1.0`.
+
+Despite that Node 6 is officially reached the End-of-Life, please, notice that it still can have maintenance updates, so it is required for the agent to get the latest binaries. You can check the currently existing Node versions [here](https://nodejs.org/dist/index.tab).
```
Comment (max-zaytsev) [inline docs/noderunner.md:14]: This is something not very readable - https://nodejs.org/dist/index.tab

Can we use https://nodejs.org/dist/ here as well?

File: docs/noderunner.md (hunk 1 above)
Comment (vmapetr) [reply docs/noderunner.md:48]: @max-zaytsev apparently, we don't have published NodeTaskRunnerInstaller docs yet - we can get rid of this string completely or just remove the link until we will know the specific URL to the docs.

Comment (max-zaytsev) [reply docs/noderunner.md:48]: I mean you can do it like this:
[NodeTaskRunnerInstaller task](http://) documentation [TODO: FIX LINK].

**Review by max-zaytsev — APPROVED** [review]

File: docs/noderunner.md (hunk 1 above)
Comment (vmapetr) [reply docs/noderunner.md:48]: Yep, this should work.

**Review by kirill-ivlev — APPROVED** [review]

**Review by tkasparek — APPROVED** [review]

**Pull request merged**